zcash_primitives/Cargo.toml

```diff
@@ -45,6 +45,7 @@ ripemd = { version = "0.1", optional = true }
 secp256k1 = { version = "0.21", optional = true }
 sha2 = "0.9"
 subtle = "2.2.3"
+zcash_address = { version = "0.1", path = "../components/zcash_address" }
```

Comment:
The previous approach avoided the need for this dependency interaction, which otherwise goes unused. However, at some point we will refactor our fully-parsed address types out of `zcash_client_backend` (once we're happy with them), and at that point we'd likely need this dependency, so it's fine to add now.